Makefile

```
@@ -9,6 +9,9 @@ clean:
 install:
 	npm install
 
+lint:
+	npm install && npx eslint . --max-warnings=0 || exit 0
```

Can we change this to use the npm runner? I think the following will work.

```suggestion
	npm install && npm run lint --max-warnings=0 || exit 0
```